project/threads/templates/threads/user/update_settings.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
Please move this template to `project/accounts/templates/accounts/update_settings.html`

Reply:
Ok sure will do aswell

Reply:
@BrianPainter  I have moved the forms, views and urls to accounts.

Reply:
The file still seems to be located in the `threads` app. Since the template is related to account settings, please move it to `/project/accounts/templates/accounts/update_settings.html`